src/power/power.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import DashboardPage from '../utils/DashboardPage';
+import Section from '../utils/Section';
+import { PowerSummary } from './summary';
+import { TimeDomain } from '../vendor/jx/domains';
+import { COMBOS, PLATFORMS, TESTS } from './config';
+import { selectFrom } from '../vendor/vectors';
+
+export default class Power extends React.Component {
+  render() {
+    const timeDomain = new TimeDomain({ past: '2month', interval: 'day' });
+    const suites = selectFrom(COMBOS)
+      .groupBy('suiteLabel')
+      .map(([v]) => ({ suiteId: v.suite, suiteLabel: v.suiteLabel })).toArray();
```

Comment:
You do not need to change `suite` to `suiteId`

Instead of 

      const suites = selectFrom(COMBOS)
      .groupBy('suiteLabel')
      .map(([v]) => ({ suiteId: v.suite, suiteLabel: v.suiteLabel })).toArray();

use

      const suites = selectFrom(COMBOS)
      .groupBy('suiteLabel')
      .map(first)
      .toArray();

and change the `suiteId` to `suite`